roles/edpm_ssh_known_hosts/tasks/main.yml

```diff
@@ -63,16 +63,16 @@
           {%     set enabled_host_networks = hostdata['edpm_role_networks']|default([]) %}
           {%     for network in enabled_host_networks %}
           {%       if (network ~ '_ip') in hostdata %}
-          {%         set _ = entries.append('[' ~ hostdata[network ~ '_ip'] ~ ']*') %}
+          {%         set _ = entries.append(hostdata[network ~ '_ip']) %}
           {%       endif %}
           {%       if (network ~ '_hostname') in hostdata %}
-          {%         set _ = entries.append('[' ~ hostdata[network ~ '_hostname'] ~ ']*') %}
+          {%         set _ = entries.append(hostdata[network ~ '_hostname']) %}
           {%       endif %}
           {%     endfor %}
```

Comment:
Why remove the `*`? That would allow us to ssh using fqdn when the entries are something like...

`edpm-compute-1* ssh-rsa .....`

Reply:
as far as I see we already generate hostname and canonica_hostname as an alternative name for the given hostkey. So I assumed that the name globbing is not needed as the fqdn will be added explicitly to the known_hosts file

Reply:
We don't have `canonical_hostname` in host vars atm and that entry is not added. Though it could add that var in inventory, I would have left the name globbing.

Reply:
I can add back the globbing selectively to the hostname case but having it at the IP or the canonical_hostname is an error.

Reply:
https://github.com/openstack-k8s-operators/edpm-ansible/pull/429